javaScript/conditionals.js

```diff
@@ -0,0 +1,19 @@
+var str = "";
+    i = 0
+
+do {
+    i++;
+    str +=i;
+}
+while (i < 116)
+
+console.log(str);
+for (var str = ""; str < 116; str+=i) {
+if (str / 3 === 0) { 
+    console.log("software");
+    } if (str / 5 === 0) {
```

Comment:
Write the next statement on the next line. Also you dont have to keep repeating "if", you can use "else if"